migrations/V41__Add_Has_Logged_In_To_User_Account.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE user_account
+ADD COLUMN has_logged_in BOOLEAN DEFAULT FALSE; --ADD Column
+
+
+UPDATE user_account --Update missing information
+SET has_logged_in = FALSE
+WHERE has_logged_in IS NULL;
+
+ALTER TABLE user_account
+ALTER COLUMN has_logged_in SET NOT NULL; -- Add not null constraint
```

Comment:
You could do this which would avoid an unsafe data migration:
```suggestion
ALTER TABLE user_account
ADD COLUMN has_logged_in BOOLEAN NOT NULL DEFAULT FALSE;
```

Reply:
Unsafe in the sense that rolling back migrations that contain data backfill is not really possible. Where rolling back an ADD COLUMN only will just delete it.